packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -24,22 +24,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
 
-    setupFiles: [
-      isEjecting
-        ? 'react-app-polyfill/jsdom'
-        : require.resolve('react-app-polyfill/jsdom'),
-    ],
+    setupFiles: [require.resolve('react-app-polyfill/jsdom')],
 
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
       '<rootDir>/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}',
     ],
-    testEnvironment: 'jsdom',
+    testEnvironment: require.resolve('jest-environment-jsdom-fourteen'),
     testURL: 'http://localhost',
```

Comment:
This is the default now (since some minor release of 23)

Reply:
Maybe GitHub is tricking me but I thought the default version of jsdom was 11.x?

Reply:
It is indeed tricking you, I commented on this line 😀 

>testURL: 'http://localhost',